native/packages/react-native-bpk-component-alert/stories.js

```diff
@@ -0,0 +1,121 @@
+    [
+      {
+        text: 'Ask me later',
+        onPress: () => action('Ask me later pressed'),
+      },
+      {
+        text: 'No',
+        onPress: () => action('negative button press'),
+        style: 'cancel',
+      },
+      { text: 'Yes', onPress: () => action('positive button press') },
+    ],
+    { cancelable: false },
+  );
+};
+
+const showCancelableAlert = () => {
+  BpkAlert.alert(
+    'Newsletter Signup',
+    'Do you wish to be notified by our newsletter for new offerings and product features',
+    [
+      {
+        text: 'No',
+        onPress: () => action('negative button press'),
+        style: 'cancel',
+      },
+      { text: 'Yes', onPress: () => action('positive button press') },
+    ],
+    { cancelable: true },
+  );
+};
+
+const showOverflowingAlert = () => {
+  BpkAlert.alert(
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit \n ' +
+      'Suspendisse a massa ac turpis suscipit varius. Quisque ac luctus diam.',
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit.\n ' +
+      'Suspendisse a massa ac turpis suscipit varius. Quisque ac luctus diam.,\n ' +
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit.\n ' +
+      'Suspendisse a massa ac turpis suscipit varius.\n ' +
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit.\n ' +
+      'Suspendisse a massa ac turpis suscipit varius.\n ' +
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit.\n ' +
+      'Suspendisse a massa ac turpis suscipit varius.\n ' +
+      'Quisque ac luctus diam. Morbi odio ligula, placerat sagittis nulla et,\n ' +
+      'dapibus ullamcorper urna.\n ' +
+      'Vestibulum pretium enim turpis, vel commodo erat vehicula vel. \n ' +
+      'Quisque ac luctus diam. Morbi odio ligula, placerat sagittis nulla et,\n ' +
+      'dapibus ullamcorper urna.\n ' +
+      'Vestibulum pretium enim turpis, vel commodo erat vehicula vel. \n ' +
+      'Quisque ac luctus diam. Morbi odio ligula, placerat sagittis nulla et,\n ' +
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit.\n ' +
+      'Suspendisse a massa ac turpis suscipit varius.\n ' +
+      'Quisque ac luctus diam. Morbi odio ligula, placerat sagittis nulla et,\n ' +
+      'dapibus ullamcorper urna.\n ' +
+      'Vestibulum pretium enim turpis, vel commodo erat vehicula vel. \n ' +
+      'Quisque ac luctus diam. Morbi odio ligula, placerat sagittis nulla et,\n ' +
+      'dapibus ullamcorper urna.\n ' +
+      'Vestibulum pretium enim turpis, vel commodo erat vehicula vel. \n ' +
+      'Quisque ac luctus diam. Morbi odio ligula, placerat sagittis nulla et,\n ' +
+      'dapibus ullamcorper urna.',
+    [
+      {
+        text: 'No',
+        onPress: () => action('negative button press'),
+        style: 'cancel',
+      },
+      { text: 'Yes', onPress: () => action('positive button press') },
+    ],
+    { cancelable: true },
+  );
+};
+
```

Comment:
We don't have an example of a [destructive alert for iOS](https://facebook.github.io/react-native/docs/alert.html#ios), I think this should also be included in storybook and screenshots.

Reply:
![img_1119](https://user-images.githubusercontent.com/73652/40534796-159fd95a-5fff-11e8-8852-4862e31dcf16.jpg)

It shows it in red - I think we should have an example that shows each of the styles, like in the above screenshot.